CHANGELOG.md

```diff
@@ -1169,7 +1177,8 @@ Initial release
 
 
 
-[Unreleased]: https://github.com/python-poetry/poetry/compare/1.1.12...1.1
+[Unreleased]: https://github.com/python-poetry/poetry/compare/1.1.13...1.1
+[1.1.13]: https://github.com/python-poetry/poetry/compare/1.1.13
```

Comment:
This should probably be a link the the tag, rather than a compare link, right? Apologies if my comment is unsollicited :)

```suggestion
[1.1.13]: https://github.com/python-poetry/poetry/releases/tag/1.1.13
```

Reply:
:+1: